pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -360,13 +375,20 @@ func (cm *containerManagerImpl) setupNode() error {
 
 	// Setup top level qos containers only if CgroupsPerQOS flag is specified as true
 	if cm.NodeConfig.CgroupsPerQOS {
-		qosContainersInfo, err := InitQOS(cm.NodeConfig.CgroupDriver, cm.NodeConfig.CgroupRoot, cm.subsystems)
+		qosContainersInfo, err := InitQOS(cm.NodeConfig.CgroupDriver, cm.cgroupRoot, cm.subsystems)
 		if err != nil {
 			return fmt.Errorf("failed to initialise top level QOS containers: %v", err)
 		}
 		cm.qosContainers = qosContainersInfo
 	}
 
+	// Enforce Node Allocatable (if required)
+	if err := enforceNodeAllocatableCgroups(cm.NodeConfig.NodeAllocatableConfig,
```

Comment:
fwiw, i like the model @sjenning had created where we end up with a localized interface per tier in hierarchy.  so he had a qos container manager interface that handled things at that tier, i could see us doing the same for the node allocatable tier.  makes it clear how we can keep any update logic localized.  not a must do for this pr or anything.